lib/pullhub/broadcasters/pull_requests_broadcaster.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Pullhub.PullRequestsBroadcaster do
+  alias Pullhub.Repo
+  alias Pullhub.User
+  alias Pullhub.Repository
+  alias Pullhub.StoredPullRequests
+
+  def broadcast(repositories, %User{} = user) do
+    repositories
+    |> prepare_for_broadcasting
+    |> perform_broadcast(user)
+  end
+
+  defp prepare_for_broadcasting(repositories) do
+    Enum.map(repositories, fn(repo) ->
+      StoredPullRequests.pull_requests(repo)
+      |> Repo.all
```

Comment:
Der er ikke noget User.all. Man går altid igennem et Repo. Det er forvirrende da vi også har kaldt de repositories vi persistere for Repository. Det skal helt sikkert ændres :)

Men man laver en query og sender den til Repo.all, Repo.get, Repo.update eller Repo.create (eller lign. funktioner)